server/notebooks/templates/notebook.html

```diff
@@ -6,7 +6,15 @@
 <script id="jsmd" type="text/jsmd">
 {{ jsmd|safe }}
 </script>
-<div id="page"></div>
+<div id="panes-container">
+  <iframe
+    id="eval-frame"
+    src="https:/extremely-alpha.iodide.app/iodide.eval-frame.dev.html"
```

Comment:
You could turn this into a template parameter, as we do with `{{ jsmd|safe }}` above. 

https://docs.djangoproject.com/en/2.1/topics/templates/#the-django-template-language